src/operator/convolution-inl.h

```diff
@@ -1,8 +1,8 @@
 /*!
- * Copyright (c) 2015 by Contributors
+ * Copyright (c) 2017 by Contributors
  * \file convolution-inl.h
- * \brief
- * \author Bing Xu
+ * \brief Ref: https://www.zhihu.com/question/28385679
```

Comment:
yeah. Point to yangqing's note instead: https://link.zhihu.com/?target=https%3A//github.com/Yangqing/caffe/wiki/Convolution-in-Caffe%3A-a-memo